cassandra/deserializers.pyx

```diff
@@ -127,8 +127,8 @@ cdef class DesCounterColumnType(DesLongType):
 
 cdef class DesDateType(Deserializer):
     cdef deserialize(self, Buffer *buf, int protocol_version):
-        cdef double timestamp = unpack_num[int64_t](buf) / 1000.0
```

Comment:
I'm a bit surprised by this since they should both be IEEE-754 doubles. Are you sure it's not just a print formatting issue? I don't want to make a science project out of it... just trying to understand, and make sure we're doing the same thing in both cython and python sides.